nss-cohort-36/bangazon-workforce-mgt-virtual-magicians
# Cb computer

# Description
User can now add a computer at  http://localhost:8000/computer/form
Closes #[Enter issue #]
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. Ensure that connection.py contains the appropriate absolute path for your machine to test locally
1. `git checkout master`
1. `git fetch --all`
1. `git checkout cb-computer`
1. Go to http://localhost:8000/computer/form and try to add a computer. If it added successfully the new computer should appear in the list.
# Checklist:
